Tests/AdyenSwiftUITests/AdyenSwiftUITests.swift

```diff
@@ -15,7 +15,6 @@ class AdyenSwiftUITests: XCTestCase {
     override func setUp() {
         super.setUp()
         app = XCUIApplication()
-        app.launchArguments = ["-UITests"]
```

Comment:
why is this deleted?

Reply:
I have injected it on [project level](https://github.com/Adyen/adyen-ios/pull/427/files#diff-74c105d362dd8cd9fc6b8ae1c32803c6134ace094ca58d05464ab88c399cf529R42)

Reply:
👍